src/components/admin/event/AdminPickupEvent/AdminPickupEventForm.tsx

```diff
@@ -245,13 +277,38 @@ const AdminPickupEventForm = ({ mode, defaultData = {}, token, upcomingEvents }:
       <div className={style.submitButtons}>
         {mode === 'edit' ? (
           <>
-            <Button submit disabled={loading}>
+            <Button submit disabled={loading} className={style.expandButton}>
               Save changes
             </Button>
-            <Button onClick={resetForm} disabled={loading} destructive>
+            <Button
+              onClick={() => completePickupEvent(uuid ?? '', token)}
+              disabled={loading}
+              className={style.expandButton}
+            >
+              Complete pickup event
+            </Button>
+            <Button
+              onClick={resetForm}
+              disabled={loading}
+              destructive
+              className={style.expandButton}
+            >
               Discard changes
             </Button>
```

Comment:
I think you should swap the complete and discard buttons so the buttons are grouped by what they do. I think grouping it that way might be better than keeping the blues and reds separate, as it currently is

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/15746db7-b911-4cf3-8373-ebe61e3cc04e)